gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/EcsClusterService.groovy

```diff
@@ -15,9 +14,7 @@ class EcsClusterService {
     this.clouddriver = clouddriver
   }
 
-  List getEcsClusters(String account, String region) {
-    HystrixFactory.newListCommand("pipelines", "updatePipeline") {
-      clouddriver.getEcsClusters(account, region)
-    } execute()
+  List getAllEcsClusters() {
```

Comment:
Please keep using Hystrix.  Hystrix is very important for service resiliency

Reply:
Done.